framework/src/org/apache/cordova/CordovaInterface.java

```diff
@@ -22,6 +22,7 @@ Licensed to the Apache Software Foundation (ASF) under one
 import android.content.Intent;
 
 import org.apache.cordova.CordovaPlugin;
+import org.json.JSONObject;
```

Comment:
It looks like there were no real changes to this file. Was this just leftover from a previous iteration?